pkg/refproto/ssl_game_event.proto

```diff
@@ -287,6 +301,8 @@ message GameEvent {
         optional uint32 victim = 4;
         // the location of the contact point between both bots
         optional Location location = 3;
+        // the location of the ball at the moment when this foul occurred
```

Comment:
Depending on the definition of what is meant by "when the foul occurs", this location will be exactly the same as the contact point. But I guess it is easier to have this here for implementation reasons

Reply:
Nevermind, wrong context, ignore this comment